src/ado2gh/Services/ReposCsvGeneratorService.cs

```diff
@@ -1,33 +1,40 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace OctoshiftCLI.AdoToGithub
 {
     public class ReposCsvGeneratorService
     {
-        public virtual async Task<string> Generate(AdoApi ado, IDictionary<string, IDictionary<string, IDictionary<string, IEnumerable<string>>>> pipelines)
+        private readonly AdoInspectorService _adoInspectorService;
+
+        public ReposCsvGeneratorService(AdoInspectorService adoInspectorService)
+        {
+            _adoInspectorService = adoInspectorService;
+        }
+
+        public virtual async Task<string> Generate(AdoApi adoApi)
```

Comment:
`AdoApi` is a dependency and should be injected.

Reply:
AdoApi is not injectable